### ISSUE TITLE
Add Web support

### ISSUE THREAD

**Issue by likern** [breeffy/react-native-calendar#34]
Support working `react-native-calendar` on web by using [react-native-web](https://github.com/necolas/react-native-web)

**Comment by nandorojo** [comment]
A few issues I'm facing on Web:

1. `export { Calendar, CalendarProps } from './Calendar'` should be changed to `export type { CalendarProps } from './Calendar`

Currently I'm seeing this issue:

<img width="547" alt="Screen Shot 2021-04-15 at 12 25 05 PM" src="https://user-images.githubusercontent.com/13172299/114904323-bd10d280-9de5-11eb-942d-e5638aceb27b.png">

For some reason, the JS module is exporting `CalendarProps` which doesn't exist. Not sure why that's happening, I haven't seen that before.

Similarly:

```sh
Attempted import error: 'SvgCircleProps' is not exported from './Circle'.
```

<img width="850" alt="Screen Shot 2021-04-15 at 12 27 15 PM" src="https://user-images.githubusercontent.com/13172299/114904509-ef223480-9de5-11eb-8dbd-d1d475d47d8b.png">

The `module` folder is exporting these TS types as JS variables.

3. After fixing those issues, I did get it to render on web. However, it doesn't actually let me swipe and the formatting is off:

<img width="1887" alt="Screen Shot 2021-04-15 at 12 28 34 PM" src="https://user-images.githubusercontent.com/13172299/114904702-1e38a600-9de6-11eb-9d58-730b19eeffa8.png">

react-native-gesture-handler works on web, so I'm not exactly sure why the gestures aren't.

Thanks for the web support goal!

**Comment by likern** [comment]
@nandorojo Not bad, not bad 😸  Honestly, it looks even better than I was expected 🤣 
Could you share your `Web` support, so I can run it and see the current state (with described bugs).

**Comment by nandorojo** [comment]
Yup, I installed it in my app and just ran the minimal example from the readme and gave it a black background. I would make a snack but they aren't adding reanimated 2 support until later this week.
